service-deployment/bootstrap/grafana/grafana-configuration/pai-nodeview-dashboard.json.template

```diff
@@ -642,7 +642,7 @@
               "decimals": 0,
               "link": true,
               "linkTooltip": "Click to view usage of each GPU",
-              "linkUrl": "{{ clusterinfo['grafanainfo']['grafana_url'] }}:{{ clusterinfo['grafanainfo']['grafana_port'] }}/dashboard/script/gpu.js?orgId=1&host=$__cell_2&rows=$__cell_4",
+              "linkUrl": "/dashboard/script/gpu.js?orgId=1&host=$__cell_2&rows=$__cell_4",
```

Comment:
@YanjieGao : Please use this relative URL, otherwise Pylon won't be able to handle this link.